### ISSUE TITLE
Enhancement request - Could look if you can implement plus logic into Megaminer ?

### ISSUE THREAD

**Issue by jrafaelmdo** [tutulino/Megaminer#358]
Hi play with nemosminer for few days and really like this "new" plus logic, its very stable and fairly resistant to pumps/dumps, also estimated earnings are very accurate.

Quote from nemosminer:

> Uses calculations based on 24hractual and currentestimate ahashpool prices to get more realistic estimate.
>   Includes some trust index based on past 1hr currentestimate variation from 24hr.
>   AND is NOT sensible to spikes.
>   This shows less switching than following Current Estimate and more switching that following the 24hr Actual.
>   Better profitability.

https://github.com/nemosminer/NemosMiner-v2.5.2-windows

Thanks.

**Comment by VukaKun** [comment]
Yes, please.

**Comment by baktery** [comment]
This suggestion sounds very good.

I saw that several times Megaminer was affected by some spikes. In order to prevent it I had to disable the algorithms producing spikes, it would be better to detect and ignore the spikes automatically.

**Comment by cap3r5** [comment]
The plus logic has been a game changer. I have am running 2 identical rigs with 10 1080's running each. I am getting around 30-40% more profits on the nemos miner one. I will be switching to nemos until plus logic is added to megaminer or a better option comes along.

**Comment by tutulino** [comment]
@cap3r5 what pool?

**Comment by cap3r5** [comment]
I've been mining at zergpool with autoconvert to LTC and compared the two pools over a period of about 4 days. It was significantly better throughout that time to use the plus logic at zergpool for my 1080 rig at least.

**Comment by podlbo** [comment]
Agree, i did test Nemos for PLUS logic and it does works realy well. My profiti is beter in in day of testing...

**Comment by tutulino** [comment]
I want to see a comparative of:

- Same hashrate machines
- Same period
- 24h mode

When I tested Current Mode and 24h mode, sometimes is best one, sometimes other, Plus logic is a intermediate point between that , I dont think always be better than 24h mode.

I Could add easily a mixed mode with price based in current and 24h, for example (080*24hprice+0*20*CurrentPrice), I think this works as Plus mode, but I can ensure this will be more profitable.

**Comment by podlbo** [comment]
It does somehow calculates  24 hour actual, current estiamted value and 24 hour estimated value to select best option... It does also calculate pool "thrust" level - so it checks what you shoud theoreticly payed and what you actualy get (simmilar is already in MM config file @@POOLPROFITFACTOR). It is realy intermediate pont between bolth worlds...
